example/reader_util.h

```diff
@@ -234,6 +235,12 @@ typedef struct ndpi_flow_info {
       char hostname[48];
       char fqdn[48];
     } softether;
+    struct {
+      char identity_uuid[36];
+      char machine[48];
+      char platform[48];
+      char services[64];
+    } tivoconnect;
```

Comment:
Could we reduce the size of these fields a little bit?
Right now, the size of `ndpi_flow_info->protos` is 168 (on x86_64; because of tls); with this change it will be 196 (because of tivoconnect).
If possible, I would prefer to avoid increasing (again...) the overall size of `struct ndpi_flow_info`